test/config/setup.ts

```diff
@@ -9,8 +9,9 @@ const EXTENSION_PATH = path.join(__dirname, '..', '..', '/dist')
 module.exports = async () => {
   console.log('Setup Puppeteer')
   const browser = await puppeteer.launch({
+    executablePath: process.env.PUPPETEER_EXEC_PATH, // set by docker container
     headless: false, // extensions only supported in full chrome.
-    args: [`--disable-extensions-except=${EXTENSION_PATH}`, `--load-extension=${EXTENSION_PATH}`],
+    args: [`--disable-extensions-except=${EXTENSION_PATH}`, `--load-extension=${EXTENSION_PATH}`, '--no-sandbox'],
```

Comment:
why did you add `--no-sandbox`?

Reply:
Pretty much just to see if it is also required and can solve our issue. Removing it now.